### ISSUE TITLE
False positive when app's name is "components"

### ISSUE THREAD

**Issue by marcoow** [ember-template-lint/ember-template-lint#761]
I just discovered there's a false positive when the app's name is _"components"_:

```
components/templates/application.hbs
  12:0  error  Unexpected {{outlet}} usage. Only use {{outlet}} within a route template.  no-outlet-outside-routes
```

This is actually the application route's template but the app happens to be named "components".

Reproduction available here: https://github.com/simplabs/ember-workshop/tree/master/examples/components

**Comment by Radioreve** [comment]
I just found out that I had the exact same problem, with an app named `ember-components`. In this [PR](https://github.com/ember-template-lint/ember-template-lint/pull/359/files), the last test to know if it's a route template is :
```
moduleName.indexOf('components/') === -1 // classic component
```
Unfortunately, every route in our application will evaluate this to `false`, since `components/` will always be part of every route name. If you create more routes, you'll see this linter error many times.

As will also evaluate to `false` for subroutes names `components/` as it seems to be the case in [this issue](https://github.com/ember-template-lint/ember-template-lint/issues/700) too.
